components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3516,4 +3516,9 @@ bool RewardsServiceImpl::IsWalletInitialized() {
   return is_wallet_initialized_;
 }
 
+void RewardsServiceImpl::SetTestResponseCallback(
```

Comment:
functions which are only used for testing should be prefixed with `ForTesting` according to Google Test documentation

Reply:
done